google-maps/README.md

```diff
@@ -24,6 +24,12 @@ Read about [Configuring `Info.plist`](https://capacitorjs.com/docs/ios/configura
 
 > The Google Maps SDK currently does not support running on simulators using the new M1-based Macbooks. This is a [known and acknowledged issue](https://developers.google.com/maps/faq#arm-based-macs) and requires a fix from Google. If you are developing on a M1 Macbook, building and running on physical devices is still supported and is the recommended approach.
```

Comment:
Maybe change this section, since Google Maps SDK does support M1 based Macbooks now, it's just `Google-Maps-iOS-Utils` what doesn't support it yet.